topics/server-opentelemetry.md

```diff
@@ -0,0 +1,327 @@
+[//]: # (title: Distributed tracing with OpenTelemetry in Ktor Server)
+
+<show-structure for="chapter" depth="2"/>
+<primary-label ref="server-plugin"/>
+<var name="plugin_name" value="KtorServerTelemetry"/>
+<var name="package_name" value="io.opentelemetry.instrumentation"/>
+<var name="artifact_name" value="opentelemetry-ktor-3.0"/>
+
+<tldr>
+<p>
+<b>Required dependencies</b>: <code>io.opentelemetry.instrumentation:opentelemetry-ktor-3.0</code>
+</p>
+<var name="example_name" value="opentelemetry"/>
+<include from="lib.topic" element-id="download_example"/>
+</tldr>
+
+<snippet id="opentelemetry-description">
+
+Ktor integrates with [OpenTelemetry](https://opentelemetry.io/) — an open-source observability framework for collecting
+telemetry data such as traces, metrics, and logs. It provides a standard way to instrument applications and export data
+to monitoring and observability tools like Grafana or Jaeger.
+
+</snippet>
+
+The `%plugin_name%` plugin enables distributed tracing of incoming HTTP requests in a Ktor server application. It
+automatically creates spans containing route, HTTP method, and status code information, extracts existing trace context
+from incoming request headers, and allows customization of span names, attributes, and span kinds.
```

Comment:
Should we link the definition of "span"? Maybe not, if our primary audience is OpenTelemetry users looking for a server, but also maybe yes, if our audience is Ktor users looking for a telemetry framework.